apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -103,10 +113,24 @@ schema() ->
                 ?IMPORTANCE_MEDIUM,
                 ?DESC(shared_subs),
                 #{}
+            )},
+        {mq_states,
+            db_schema(
+                [builtin_raft, builtin_local],
+                ?IMPORTANCE_MEDIUM,
+                ?DESC(mq_states),
+                #{}
+            )},
+        {mq_messages,
+            db_schema(
+                [builtin_raft, builtin_local],
+                ?IMPORTANCE_MEDIUM,
+                ?DESC(mq_messages),
+                #{}
             )}
```

Comment:
is it possible to configure builtin_raft for one and builtin_local for another ?